src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1849,6 +1850,11 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
             $container->removeDefinition('serializer.normalizer.mime_message');
         }
 
+        // Backward compatibility with Serializer < 7.2
```

Comment:
I know that generally the `// BC layer for ...` format for this kind of comment is used. Maybe this comment should use the same format for easier searching?